external/header.php

```diff
@@ -112,17 +113,17 @@ public static function __callstatic($name, $arguments)
 unset($domain);
 
 //Display warning if extension not available
-if (extension_loaded('xhprof') && $_xhprof['doprofile'] === true) {
+if (( extension_loaded('tideways') || extension_loaded('xhprof') ) && $_xhprof['doprofile'] === true) {
     include_once dirname(__FILE__) . '/../xhprof_lib/utils/xhprof_lib.php';
     include_once dirname(__FILE__) . '/../xhprof_lib/utils/xhprof_runs.php';
     if (isset($ignoredFunctions) && is_array($ignoredFunctions) && !empty($ignoredFunctions)) {
-        xhprof_enable(XHPROF_FLAGS_CPU + XHPROF_FLAGS_MEMORY, array('ignored_functions' => $ignoredFunctions));
+        (extension_loaded('tideways') ? tideways_enable(TIDEWAYS_FLAGS_CPU + TIDEWAYS_FLAGS_MEMORY, array('ignored_functions' => $ignoredFunctions)) : xhprof_enable(XHPROF_FLAGS_CPU + XHPROF_FLAGS_MEMORY, array('ignored_functions' => $ignoredFunctions)) );
```

Comment:
Yep, this one is ugly dirty, it's a fact, but I didn't have the time to do it differently, although I kept thinking about it when changing the code.

The thing is that it might not be necessary to create a full plugin, as only the naming is different from xhprof/tideways/uprofiler... 
Everything else is the same, the base features are the same in all of them, others just have more features , like tideways, has a paid remote UI (kind of like blackfire)

We could do this differently without complicating things too.